docs/en/latest/plugins/opentelemetry.md

```diff
@@ -124,7 +151,7 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 
 ## Disable Plugin
```

Comment:
@navendu-pottekkat I'm not sure if we should use `Disabling Plugin` or using `Disable Plugin` 😅 I would prefer `Disable Plugin` because it's shorter and easy to read, but I notice some other products' documentations are using `xxx-doing`.

Reply:
Disable Plugin is good IMO as well.